pkg/collector/corechecks/network/ntp.go

```diff
@@ -156,6 +170,35 @@ func (c *NTPCheck) Run() error {
 	return nil
 }
 
+func (c *NTPCheck) queryOffset() (float64, error) {
+	offsets := []float64{}
+
+	for _, host := range c.cfg.instance.Hosts {
+		response, err := ntpQuery(host, c.cfg.instance.Version)
+		if err != nil {
+			log.Infof("There was an error querying the ntp host %s: %s", host, err)
+		} else {
+			offsets = append(offsets, response.ClockOffset.Seconds())
+		}
+	}
+
+	if len(offsets) == 0 {
+		return .0, fmt.Errorf("Failed to get clock offset from any ntp host")
+	}
+
+	var median float64
```

Comment:
median or mode? To be honest, with 4 values it probably doesn't really matter much.

Reply:
Using median or rounding and using mode both seems fine to me